### PR TITLE
Fixing context readmes - extra spaces, tabs and formatting

### PR DESCRIPTION
**Pull request by roysagi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/demisto-sdk/pull/1539

## Description
Fixing formatting issues identified by fixed sdk validation

### CODE REVIEW

File: Packs/Smokescreen_IllusionBLACK/ReleaseNotes/1_0_9.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Smokescreen IllusionBLACK
+- Documentation and metadata improvements.
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
```
Comment (roysagi) [inline Packs/Smokescreen_IllusionBLACK/ReleaseNotes/1_0_9.md:5]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.9.6.22912*.
```

File: Packs/Infinipoint/ReleaseNotes/1_0_7.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Infinipoint
+- Documentation and metadata improvements.
+- Updated the Docker image to: *demisto/pyjwt3:1.0.0.16907*.
```
Comment (roysagi) [inline Packs/Infinipoint/ReleaseNotes/1_0_7.md:5]:
```suggestion
- Updated the Docker image to: *demisto/pyjwt3:1.0.0.23674*.
```

File: Packs/CrowdStrikeFalconX/ReleaseNotes/1_1_4.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CrowdStrike Falcon X
+- Documentation and metadata improvements.
+- Updated the Docker image to: *demisto/python3:3.9.5.21272*.
```
Comment (roysagi) [inline Packs/CrowdStrikeFalconX/ReleaseNotes/1_1_4.md:5]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.9.6.22912*.
```

File: Packs/CiscoStealthwatch/ReleaseNotes/1_0_2.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Cisco Secure Network Analytics (Stealthwatch)
+- Documentation and metadata improvements.
+- Updated the Docker image to: *demisto/python3:3.9.4.18682*.
```
Comment (roysagi) [inline Packs/CiscoStealthwatch/ReleaseNotes/1_0_2.md:5]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.9.6.22912*.
```

File: Packs/AWS-Lambda/ReleaseNotes/1_2_2.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### AWS - Lambda
+- Documentation and metadata improvements.
+- Updated the Docker image to: *demisto/boto3py3:1.0.0.20516*.
```
Comment (roysagi) [inline Packs/AWS-Lambda/ReleaseNotes/1_2_2.md:5]:
```suggestion
- Updated the Docker image to: *demisto/boto3py3:1.0.0.23735*.
```

**Review by bakatzir — APPROVED** [review]

File: Packs/AWS-IAM/ReleaseNotes/1_1_1.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### AWS - IAM
+- Documentation and metadata improvements.
+- Updated the Docker image to: *demisto/boto3:2.0.0.20513*.
```
Comment (roysagi) [inline Packs/AWS-IAM/ReleaseNotes/1_1_1.md:5]:
```suggestion
- Updated the Docker image to: *demisto/boto3:2.0.0.23135*.
```

**Comment by roysagi** [comment]
Validate currently fails on the following:
Will be fixed in the next SDK release:
All the RM102 and IN136 are a result of a change in the regex for these commands
 - Packs/AWS-IAM/Integrations/AWS-IAM/README.md - [RM102]
 - Packs/AWS-IAM/Integrations/AWS-IAM/AWS-IAM.yml - [IN136]
 - Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/README.md - [RM102]
 - Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/cisco_stealthwatch.yml - [IN136]
 - Packs/AWS-Lambda/Integrations/AWS-Lambda/README.md - [RM102]
 - Packs/AWS-Lambda/Integrations/AWS-Lambda/AWS-Lambda.yml - [IN136]
 - Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/README.md - [RM102]
 - Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.yml - [IN136]
 - Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/cisco_stealthwatch.yml - [ST100] - RN validator is currently using  regex as part of the flow which looks for the same name for the yml file and package dir name - this will no loner be the case.

The following are BC breaks due to spaces being deleted in the end of context paths:
 - Packs/AWS-IAM/Integrations/AWS-IAM/AWS-IAM.yml - [BC102]
 - Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/cisco_stealthwatch.yml - [BC102]
 - Packs/AWS-Lambda/Integrations/AWS-Lambda/AWS-Lambda.yml - [BC102]
 - Packs/Smokescreen_IllusionBLACK/Integrations/Smokescreen_IllusionBLACK/Smokescreen_IllusionBLACK.yml - [BC102]
 - Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.yml - [BC102]

Lastly these changed integrations do not have tests:
 - Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/cisco_stealthwatch.yml - [IN140]
 - Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.yml - [IN140]

This PR will require a force merge once new SDK version rolls out

**Review by yaakovi — APPROVED** [review]

**Comment by Itay4** [comment]
build fails on approved validations
merging

**Pull request merged**